### ISSUE TITLE
M1 Txs and data

### ISSUE THREAD

**Issue by 0xPrimata** [movemntdev/M1#86]
Petra and other wallets are unable to Transact and/or display data correctly.

![image](https://github.com/movemntdev/M1/assets/6299375/9b3f4210-4bd8-4671-ba81-1ebfe1c7f439)
![image](https://github.com/movemntdev/M1/assets/6299375/3c82bbd7-4a97-4971-8e3d-acd77394d4ea)

**Comment by l-monninger** [comment]
This is new AFAIK. But, I think it underscores the importance of https://github.com/movemntdev/M1/issues/74

**Comment by danMove** [comment]
I don't think this is an issue with Chain specifically. I encounter the same problem when using the Petra Devnet network. We should wait for the wallet developers to fix this problem.

https://github.com/movemntdev/M1/assets/125717416/4456b339-3aa0-45a9-88c6-043367472302
